client/src/pages/Profile.js

```diff
@@ -3,17 +3,43 @@ import CandidateDetails from "../components/CandidateDetails";
 import Separator from "../components/Separator";
 import ProfileDetails from "../components/ProfileDetails";
 import styled from "@emotion/styled";
+import { getCandidateDetails } from "../components/getCandidateDetails";
 
 const PageWrapper = styled.div`
   display: flex;
 `;
 
-export default function Profile() {
+export default function Profile(candId) {
+  const [profile, setProfile] = React.useState([]);
+
+  async function refreshProfile() {
+    const foundProfile = await getCandidateDetails(candId);
+    setProfile(foundProfile);
+    return profile;
+  }
+
+  React.useEffect(() => {
+    refreshProfile();
+    // eslint-disable-next-line
```

Comment:
Why do you have to disable eslint here? If it warns because of the `canId`, add it to the useEffect array `[candid] `